gooddata-sdk/docs/services/catalog-workspace-content.rst

```diff
@@ -69,13 +69,13 @@ The *gooddata_sdk.catalog_workspace_content* supports the following declarative
 
     Returns *CatalogDeclarativeModel*.
 
-    Retrieve a logical model layout. On CatalogDeclarativeModel user can call ``modify_mapped_data_source(data_source_mapping: dict)`` method, which substitues data source id in datasets.
+    Retrieve a logical model layout. On CatalogDeclarativeModel user can call ``modify_mapped_data_source(data_source_mapping: dict)`` method, which substitutes data source id in datasets.
 
-* ``put_declarative_ldm(workspace_id: str, ldm: CatalogDeclarativeModel)``
+* ``put_declarative_ldm(workspace_id: str, ldm: CatalogDeclarativeModel, validator: Optional[DataSourceValidator])``
 
-    Set a logical model layout.
+    Put a logical data model to a given workspace. You can pass an additional validator parameter, which checks that for every data source id in the logical data model the corresponding data source exists.
```

Comment:
Put a logical data model into a given workspace. You can pass an additional validator parameter which checks that for every data source id in the logical data model the corresponding data source exists.